lib/metrics.py

```diff
@@ -8,8 +8,8 @@
 import itertools
 import os
 
-from common import get_media, memoize, timefn
-from framereader import FrameReaders
+from .common import get_media, memoize, timefn
+from .framereader import FrameReaders
```

Comment:
It's unfortunate that this (5dd1d67bd81066ba0b7cca92afeba83c03522246) did not get included with #138 PR.  Please separate this commit into separate PR and push immediately since ./vaapi-fits is now broken without it.